admin/includes/template/templates/tplIndex.php

```diff
@@ -35,6 +35,7 @@
 <script>
 var intervalTimers = new Array();
 $(function() {
+  <?php if (isset($tplVars['widgetList']) && (sizeof($tplVars['widgetList'] == 0))) { ?> 
```

Comment:
Argh you are correct.  I will fix.  Thanks for catching that.

On Thu, Mar 12, 2015 at 1:08 PM, Chris Brown <notifications@github.com>
wrote:

> In admin/includes/template/templates/tplIndex.php
> <https://github.com/zencart/zc-v1-series/pull/344#discussion_r26322318>:
>
> > @@ -35,6 +35,7 @@
> >  <script>
> >  var intervalTimers = new Array();
> >  $(function() {
> > +  <?php if (isset($tplVars['widgetList']) && (sizeof($tplVars['widgetList'] == 0))) { ?>
>
> And, shouldn't the closing paren for sizeof be before the == 0?
>
> I suggest:
>
> <?php if (isset($tplVars['widgetList']) && sizeof($tplVars['widgetList']) == 0) { ?>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zencart/zc-v1-series/pull/344/files#r26322318>.
>